resumes/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import uuid
 
-# Create your views here.
+from django.views    import View
+from django.http     import JsonResponse
+
+from .models         import *
+from core.storages   import S3Client
+from my_settings     import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_STORAGE_BUCKET_NAME
+from needed.settings import AWS
+from users.utils     import authorization
+
+class ResumeView(View):
+    @authorization
+    def post(self, request):
+        try:
+            file           = request.FILES.get('document', None)
```

Comment:
그냥 file이 아니라 정확한 네이밍 해주세요.
```suggestion
           resume_file           = request.FILES.get('resume', None)
```